clients/drcachesim/tracer/tracer.h

```diff
@@ -254,7 +257,8 @@ has_tracing_windows()
     // We return true for a single-window -trace_for_instrs (without -retrace) setup
     // since we rely on having window numbers for the end-of-block buffer output check
     // used for a single-window transition away from tracing.
-    return op_trace_for_instrs.get_value() > 0 || op_retrace_every_instrs.get_value() > 0;
+    return op_L0_filter_until_instrs.get_value() > 0 ||
```

Comment:
What about the filetype for non-split-window all-in-one traces: do we need a new type for filtered-sometimes?  And how are the filtered regions easily identified: is some new marker needed at the start of each filtered window?

Reply:
Oh I hadn't thought of all-in-one traces. Do they have a window marker at the start of each window? I have an endpoint marker between the filtered and the full trace. The trace in this case would have {window marker, filtered trace, endpoint marker, full trace}. We will have to update raw2trace to handle the switch from full to filtered at the start of a window.

Reply:
How about updating the documentation for the filtered filetypes to make it clear such a file might transition to non-filtered if there is an endpoint marker present, and that a single-file multi-window file that starts out filtered is guaranteed to have every window start out filtered?  Then we don't put in any new marker.

Reply:
Added documentation.